openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -191,6 +196,40 @@ def test_disable_credit_requirements(self):
         self.assertEqual(len(grade_req), 1)
         self.assertEqual(grade_req[0].active, False)
 
+    def test_set_credit_requirement_status(self):
+        self.add_credit_course()
+        requirements = [
+            {
+                "namespace": "grade",
+                "name": "grade",
+                "display_name": "Grade",
+                "criteria": {
+                    "min_grade": 0.8
+                }
+            },
+            {
+                "namespace": "reverification",
+                "name": "i4x://edX/DemoX/edx-reverification-block/assessment_uuid",
+                "display_name": "Assessment 1",
+                "criteria": {}
+            }
+        ]
+
+        set_credit_requirements(self.course_key, requirements)
+        course_requirements = CreditRequirement.get_course_requirements(self.course_key)
+        self.assertEqual(len(course_requirements), 2)
+
+        requirement = get_credit_requirement(self.course_key, "grade", "grade")
```

Comment:
The `get_credit_requirement` API function is missing its own test coverage. It appears to be an integral part of the credit API, and as such the test suite should verify that it's behaving as expected.